### PR TITLE
feat(API): Implement figma attachments endpoints

### PR DESCRIPTION
**Pull request by Varpusparvi**
Screenshot 2023-09-27 at 14 29 21

- https://phrase.atlassian.net/browse/TSE-613
- https://github.com/phrase/phrase/pull/11856

### CODE REVIEW

**Comment by Varpusparvi** [comment]
Reminder to self: I will bump GO before Bumping the CLI

File: paths/figma_attachment_keys/create.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+summary: Attach the Figma attachment to a key
+description: Attach the Figma attachment to a key
+operationId: figma_attachment_key/create
```
Comment (jablan) [inline paths/figma_attachment_keys/create.yaml:4]: wouldn't it be more logical to add these operations to `figma_attachments` (or perhaps `keys`) instead?

Comment (Varpusparvi) [reply paths/figma_attachment_keys/create.yaml]: Adjusted, not exactly sure if this is what you meant 
[CR](https://github.com/phrase/openapi/pull/415/commits/f4872f5e44bec82004dfae7f99b2d3260737a6fb)

Comment (jablan) [reply paths/figma_attachment_keys/create.yaml]: yeah, but they'd also have to live on `paths/figma_attachments` IMO

Comment (jablan) [reply paths/figma_attachment_keys/create.yaml]: but I guess the API PR has already merged in phrase?

Comment (Varpusparvi) [reply paths/figma_attachment_keys/create.yaml]: Yes the API PR is merged. Can we have two `create.yaml`/`destroy.yaml` in the same folder? I could adjust this. On the API PR we agreed to not use custom named methods and instead went with second controller to have option for second create/destroy paths

**Review by jablan — APPROVED** [review]

**Pull request merged**